<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QDialog" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>12</y>
     <width>496</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/img/img/Team Logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Select Team to Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_Team_Selection"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>241</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <widget class="QLabel" name="label_NewStadium">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Enter Name of New Stadium:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_NewStadium">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_NewStadium_Capacity">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Capacity</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_NewStadium_Capacity">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_NewStadium_OpenDate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Open Date (year)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_NewStadium_Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>2020</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_NewStadium_Add">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Add New Stadium</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>171</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_SelectSouvenir">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Select Souvenir to Edit Price</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_SelectSouvenir"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_SelectSouvenir_Price">
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_SelectSouvenir_SetPrice">
        <property name="text">
         <string>Set Price</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_SelectSouvenir_Delete">
      <property name="text">
       <string>Delete Souvenir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_NewSouvenir">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Enter Name of New Souvenir:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_NewSouvenir_Name"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_NewSouvenir_Price">
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_NewSouvenir_Price">
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.250000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_NewSouvenir_Add">
      <property name="text">
       <string>Add New Souvenir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>171</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_SelectStadium">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Select Stadium:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_SelectStadium"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_SelectStadium_Capacity">
        <property name="text">
         <string>Capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_SelectStadium_SetCapacity">
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_SelectStadium_SetCapacity">
      <property name="text">
       <string>Set New Capacity</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
